sources/System/Renderer/CameraSystem.cpp

```diff
@@ -46,17 +44,17 @@ namespace BBM
 		float lowerXDist = 0;
 		float lowerZDist = 0;
 
-		for (auto &[entity, pos, _] : this->_wal.getScene()->view<PositionComponent, TagComponent<Player>>()) {
-			if (!entity.hasComponent<ControllableComponent>())
-				entity.addComponent<ControllableComponent>();
-			playerPos.emplace_back(pos.position);
+		for (auto &[player, position, _] : this->_wal.getScene()->view<PositionComponent, TagComponent<Player>>()) {
+			if (!player.hasComponent<ControllableComponent>())
+				player.addComponent<ControllableComponent>();
+			playerPos.emplace_back(position.position);
 		}
 		if (playerPos.size() == 0)
 			introAnimation(entity, true);
 		if (playerPos.size() == 1)
 			newCameraPos = playerPos[0];
-		for (int i = 0; i < playerPos.size(); i++)
-			for (int j = 0; j < playerPos.size(); j++) {
+		for (int i = 0; i < static_cast<int>(playerPos.size()); i++)
```

Comment:
Same as before, you should use std::size_t as an iterator instead of silencing the warning with a static cast.